### PR TITLE
Add php_ini type and have_config matcher.

### PR DESCRIPTION
**Pull request by RageZBla**
Hi Mizzy,

First thanks for all your work on this tool. 

Here my situation, I was trying to write some specifications for an Ubuntu server with some Nginx PHP-FPM stack. As you may know Ubuntu has specific php.ini for CLI, apache2, FPM. Sadly, you can't specify the php.ini when using php_config type so I have created my own type called php_ini.

Let me know what do you think about it, any feedback is welcome!

Thanks!

### CODE REVIEW

**Comment by mizzy** [comment]
`php_config` and `php_ini` have similar functions. It's confusing.
I'd like to avoid it.

How about extending `php_config` like this ?

```
 describe php_config('session.cache_expire', :ini => '/etc/php5/fpm/php.ini') do
    its(:value) { should eq 180 }
  end
```

**Comment by RageZBla** [comment]
Your idea is better, I'll modify my PR tonight.

**Comment by RageZBla** [comment]
Hey @mizzy,

I have updated the PR, if you have any feedback let me know.

File: lib/serverspec/type/php_config.rb [hunk 1]
```diff
@@ -1,7 +1,14 @@
 module Serverspec::Type
   class PhpConfig < Base
+    def initialize(name, options = {})
+      super(name)
+      @options = options
+    end
+
```
Comment (mizzy) [inline lib/serverspec/type/php_config.rb]: Could you change `initialize` method of `Serverspec::Type::Base`, please ?

`options` would be useful for other types.

**Comment by RageZBla** [comment]
@mizzy here you go!

**Comment by mizzy** [comment]
Thanks!

**Comment by RageZBla** [comment]
Thanks!

**Comment by mizzy** [comment]
Released as v2.26.0.

**Pull request merged**